### ISSUE TITLE
Feat: add a profile customisation button in the account settings page

### ISSUE THREAD

**Issue by charline-rageade** [tracim/tracim#4570]
## Feature description and goals

When the user goes to the account settings section, he cannot edit his profile. To facilitate the user journey, a new button can be implemented. This would directly link to the profile page and the user will be able to modify their information.

![personnalisation](https://user-images.githubusercontent.com/82377347/115852427-5bc1b280-a428-11eb-8f17-2c739d7ee02d.png)

This picture is not up to date for user information, it only illustrates the position of the button.

#3287 
#4551

**Comment by Relf-Algoo** [comment]
I was working on user's profile from an administrator point of view and I thought to myself : we need a button or link to go from the user's account to the user's profile. And lo ! there already is an issue regarding this, issue which should be brought back to memory.

**Comment by Millefeuille42** [comment]
Done
